packages/desktop/electron-builder-config.js

```diff
@@ -126,11 +138,7 @@ const prereleaseNsisOptions = {
 
 const testConfig = () => {
     const icons = getIconPaths()
-    const publish = {
-        publishAutoUpdate: false,
-    }
-
-    return merge({}, prodConfig(), icons, { appId: APP_ID }, prereleaseNsisOptions, { publish })
+    return merge({}, prodConfig(), icons, { appId: APP_ID }, prereleaseNsisOptions)
 }
 
 const build = () => {
```

Comment:
is there a reason why we use `testConfig` for `alpha` and `beta`s

Reply:
Because we have different logos and icons i thinl